changelog/unreleased/issue-2688

```diff
@@ -0,0 +1,8 @@
+Bugfix: Handle comma-separated list tags when using backup command
+
+Creating a backup by specifying comma-separated tags created a backup with this concatenated list as a tag.
+For example `restic backup --tag foo,bar` creates a new snapshot with one "foo,bar" tag instead of two tags, one "foo" and one "bar"?
+
+Simply type `[]string` to `restic.TagList` and use f.Var() to handle args. cobra did the rest!
```

Comment:
That explanation is too technical for most users. It should be enough to just state what the new behavior is, not how it's implemented.